example_test.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials_test
+
+import (
+	"context"
+	"fmt"
+
+	credentials "github.com/oras-project/oras-credentials-go"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func ExampleNewNativeStore() {
+	// The argument of NewNativeStore can be the native keychains
+	// ("wincred" for Windows, "pass" for linux and "osxkeychain" for macOS),
+	// or any program that follows the docker-credentials-helper protocol.
+	// See: https://docs.docker.com/engine/reference/commandline/login#credentials-store
+	ns := credentials.NewNativeStore("pass")
```

Comment:
Just realized that maybe we should move these comments to the `NewNativeStore`.
![image](https://user-images.githubusercontent.com/8113721/234780098-32fe823f-46c5-4fa8-95c2-2743d1851b61.png)

Reply:
moved.